src/north/CsvToHttp/CsvToHttp.schema.jsx

```diff
@@ -81,6 +81,25 @@ schema.form = {
         <p>
           The unselected keys will be omitted during the mapping.
         </p>
+        <p>
+          <b>Example 1: </b>
+          If the CSV file contains 2 headers called header1 and header2, it is possible to specify the name for each field in the
+          http request. It&lsquo;s possible to have in the body of the request a field called &quot;AfterMapping&quot;
+          which contains the value of the header1 for each row. We can do the same for each header present in the CSV file
+        </p>
+        <p>
+          <b>Example 2: </b>
+          It is possible to specify the name for each header in the http request using template string
+          (references to other header as a variable). Thanks to it we can reference header present in the CSV file as variable.
+          For it, the syntax to use is the javascript one (such as: $
+          {'{name of your header}'}
+          ).
+          With the same CSV file, it is possible to have in the request&lsquo;s body a field called header1AndHeader2 its value is the
+          concatenation of both header&lsquo;s value.
+          The &quot;Header in CSV file&quot; must be filled with the value: $
+          {'{header1} and {header2}'}
+          .
```

Comment:
Can you add a small sentence like : if a row has 'value1' in the column 'header1' and 'value2' in the column 'header2', the resulting value will be 'value1 and value2'